dogstatsd/src/dogstatsd.rs

```diff
@@ -86,6 +86,7 @@ impl DogStatsD {
     fn insert_metrics(&self, msg: Split<char>) {
         let all_valid_metrics: Vec<Metric> = msg
             .filter(|m| !m.is_empty())
+            .filter(|m| !m.starts_with("_sc|")) // exclude service checks
```

Comment:
Seems like you should also filter out events too (prefix of `_e{`) if all you want is metrics.

Reply:
https://github.com/DataDog/libdatadog/blob/e2097f7963793f54e7c515740b89b88273fd8247/dogstatsd/src/dogstatsd.rs#L90